scripts/cloudlab/setup_node.sh

```diff
@@ -58,3 +59,10 @@ if [ "$SANDBOX" == "firecracker" ]; then
     $SCRIPTS/create_devmapper.sh
 fi
 
+if [ -z "$USE_STARGZ" ]; then
+    USE_STARGZ="no-stargz"
+fi
+
+if [ "$USE_STARGZ" == "use-stargz" ]; then
+    $SCRIPTS/stargz/setup_stock_only_stargz.sh
+fi
```

Comment:
Webhook admission error probably happens because the cluster is not ready/healthy. The script in which you inserted Stargz is not the last one that is executed in the setup pipeline, so this means that the cluster has not been fully set up at the point you insert Stargz. Is this the behavior you wanted?

If you look into the quick start guide you can see that `./scripts/cluster/create_one_node_cluster.sh` is the last script in the single-node setup, whereas `./scripts/cluster/create_multinode_cluster.sh` is the last script in the multinode setup.

Reply:
That is expected. To rectify, it is not this part of the script that is failing. If you look into the [workflow file](.github/workflows/stargz_tests.yml) you will see the `./scripts/cluster/create_one_node_cluster.sh` is ran after the stargz setup.

Also, the current output of the workflow is as follows:
```bash
Run kn service apply stargz-test -f ./configs/knative_workloads/stargz-node.yaml --concurrency-target 1
Creating service 'stargz-test' in namespace 'default':

  0.192s The Route is still working to reflect the latest desired specification.
  0.305s ...
  0.346s Configuration "stargz-test" is waiting for a Revision to become ready.
  0.858s Revision "stargz-test-00001" failed with message: 0/1 nodes are available: 1 Insufficient cpu..
  0.902s Configuration "stargz-test" does not have any ready Revision.
Error: RevisionFailed: Revision "stargz-test-00001" failed with message: 0/1 nodes are available: 1 Insufficient cpu..
Run 'kn --help' for usage
```
Therefore I am assuming it is something to do with the resource management done by the scripts that setup contour.

Reply:
I am not sure how that helps, as the workflow stops after the service cannot be started and I cannot get the output of `describe`.

Reply:
@DavidBuzatu-Marian, you can set the `continue-on-error` attribute to true. More: https://www.thisdot.co/blog/how-to-retry-failed-steps-in-github-action-workflows